Atomic/atomic.py

```diff
@@ -30,6 +41,22 @@
 from docker.errors import NotFound
 
 IMAGES = []
+ATOMIC_LIBEXEC = os.environ.get('ATOMIC_LIBEXEC', '/usr/libexec/atomic')
+
+OSTREE_DOCKERIMG_PREFIX = "dockerimg/"
+SYSTEMD_UNIT_FILES_DEST = "/etc/systemd/system"
```

Comment:
no, this is only into `atomic`.  I kept two different namespaces for images that are imported directly from Docker (either registry or tarball) and those that are created directly into an OSTree commit.
In the first case we want to track every layer used to compose the image, so that it can be reused.  Each layer is mapped to an OSTree branch with the dockerimg/ prefix.

For images that are created directly as OSTree commits (as `ostree commit -b IMAGE_NAME rootfs`) there is no mapping with a Docker layer so there is no `dockerimg/` prefix.

We can "flatten" a Docker imported image by having a commit of the final rootfs, and this can be done at any stage, but doing so will lose all the advantages of tracking the Docker layers.